pyproject.toml

```diff
@@ -1,2 +1,35 @@
 [tool.black]
-target_version = ['py38']
\ No newline at end of file
+target_version = ['py38']
+
+[tool.poetry]
+name = "eq-questionnaire-validator"
+version = "0.1.0"
+description = "API for questionnaire schema validation"
+authors = ["ONSDigital"]
+
+[tool.poetry.dependencies]
+python = "^3.9"
+Flask = "^2.0.2"
+jsonpointer = "^2.2"
+structlog = "^21.5.0"
+python-dateutil = "^2.8.2"
+jsonschema = "^4.4.0"
+jsonpath-rw = "^1.4.0"
+jsonpath-rw-ext = "^1.2.2"
+requests = "^2.27.1"
+eq-translations = {git = "https://github.com/ONSDigital/eq-translations.git"}
+tqdm = "^4.62.3"
+
+[tool.poetry.dev-dependencies]
+pytest = "^6.2.5"
+flake8 = "^4.0.1"
+pylint = "^2.12.2"
+pylint-quotes = "^0.2.3"
+pytest-cov = "^3.0.0"
+pytest-xdist = "^2.5.0"
+pytest-sugar = "^0.9.4"
+black = "21.10b0"
```

Comment:
How come the packages don't match the Pipfile?. Some are sub dependencies?.

Reply:
Good spot, when eq-translations was failing, i added them manually using `poetry add <package>` . i have now used same packages as `pipefile`